src/components/QuestionEdit/index.tsx

```diff
@@ -84,6 +94,28 @@ export default function QuestionEdit({
       return 'Please select 1 correct option!';
     } else if (options.length < 2) {
       return 'At least 2 options are mandatory!';
+    } else {
+      const optionTexts = options.map((x) => x.text);
+
+      if (optionTexts.length >= 2) {
+        const el = document.createElement('div');
+
+        for (let i = 0; i < optionTexts.length - 1; i++) {
+          el.innerHTML = optionTexts[i];
+          const option1Text = el.innerText.trim().replace(/\s+/g, ' ');
```

Comment:
Can we create a helper function that does both trim + replace and returns clean string?